### ISSUE TITLE
Line chart with tooltips is broken

### ISSUE THREAD

**Issue by domoritz** [vega/vega-lite#4122]
```json
{
  "$schema": "https://vega.github.io/schema/vega-lite/v3.json",
  "description": "Stock prices of 5 Tech Companies Over Time.",
  "data": {"url": "data/stocks.csv", "format": {"type": "csv"}},
  "mark": "line",
  "encoding": {
    "x": {"field": "date", "type": "temporal", "axis": {"format": "%Y"}},
    "y": {"field": "price", "type": "quantitative"},
    "color": {"field": "symbol", "type": "nominal"},
    "tooltip": [
      {"field": "date", "type": "temporal"},
      {"field": "symbol", "type": "nominal"},
      {"field": "price", "type": "quantitative"}
    ]
  }
}
```

Somehow it has too many fields in the facet groupby (and also redundant fields).

**Comment by domoritz** [comment]
cc @invokesus do you think this is related to your facet issue?

**Comment by weiglemc** [comment]
If it helps, here's another example: https://beta.observablehq.com/d/815480fe7e6b5258
This shows a line chart with points. When tooltips are added, the lines disappear but the points remain.

**Comment by domoritz** [comment]
Sounds like the problem is that we are adding the tooltip fields to the groupby of the aggregates.

**Comment by domoritz** [comment]
I'm going to look at this now. We should fix this before 3.0.
